lib/dialects/mssql/query/mssql-querycompiler.js

```diff
@@ -479,6 +479,22 @@ class QueryCompiler_MSSQL extends QueryCompiler {
     }
     return offset;
   }
+
+  whereLike(statement) {
+    return `[${
```

Comment:
Shouldn't this need some escaping for values? I think for columns we typically use `formatter.columnize`, and for a single value `client.parameter`. See `C:\sources\node\knex\lib\query\querycompiler.js` for an example.